### PR TITLE
Saves the localized file to correct filepath

### PR DESCRIPTION
**Pull request by caylahamann**
## Summary 

Before the file path was saved to `src/i18n/content/jp/docs/some-file.mdx` when we want to save it to `src/i18n/content/jp/docs/full/filepath/some-file.mdx`. This solves this!

### CODE REVIEW

File: scripts/actions/fetch-and-deserialize.js [hunk 1]
```diff
@@ -40,15 +40,20 @@ const fetchAndDeserialize = (accessToken) => async ({ locale, fileUris }) => {
   const zipEntries = zip.getEntries();
 
   const translatedHtml = zipEntries.map((entry) => {
+    console.log(entry.entryName);
```
Comment (zstix) [inline scripts/actions/fetch-and-deserialize.js:43]: We can probably remove this `console.log`.
```suggestion
```

**Review by zstix — COMMENTED** [review]
One minor suggestion, but this looks great!

File: scripts/actions/fetch-and-deserialize.js [hunk 2]
```diff
@@ -40,15 +40,20 @@ const fetchAndDeserialize = (accessToken) => async ({ locale, fileUris }) => {
...
     return {
-      path: entry.entryName.split('/').pop().split('.').slice(0, -1), // get the last element from the folder
+      path: entry.entryName
```
Comment (jerelmiller) [inline scripts/actions/fetch-and-deserialize.js:45]: Its a bit hard to follow what this is doing. This also feels like it could be a touch brittle. I would suggest a couple changes.

1) Since you're trying to get rid of `src/jp` as part of the name, I would suggest just using a `replace`. Its a bit clearer the intention and harder to break:

```js
entry.entryName.replace(`src/${locale}`, '')
```

2) Use `path.basename` + `path.extname` to get rid of the extension. This makes it a bit clearer and ensures edge cases are handled:

```js
path.basename(entry.entryName, path.extname(path.entryName)) // => index.mdx -> index
```

Putting that altogether makes it easier to understand the purpose:

```js
const path = require('path')

const filepath = entry.entryName.replace(`src/${locale}`)

return {
  path: path.basename(filepath, path.extname(filepath))
}
```

File: scripts/actions/fetch-and-deserialize.js (hunk 1 above)
Comment (jerelmiller) [inline scripts/actions/fetch-and-deserialize.js:43]: Was this meant to be logged? If so, should this include our little `[*]` denotation?

File: scripts/actions/fetch-and-deserialize.js (hunk 2 above)
Comment (caylahamann) [reply scripts/actions/fetch-and-deserialize.js:44]: yes! I like this suggestion :)

File: scripts/actions/fetch-and-deserialize.js (hunk 1 above)
Comment (caylahamann) [reply scripts/actions/fetch-and-deserialize.js]: No it was just for testing purposes! It's removed now :)

**Review by jerelmiller — APPROVED** [review]
Shibby!
